jsk_panda_robot/jsk_panda_teleop/srv/CommandImitationAgent.srv

```diff
@@ -0,0 +1,11 @@
+string command_type  # eihter 'reset', 'act', 'get_trajectory'
```

Comment:
please add comments on messages values, like https://github.com/ros/executive_smach/blob/noetic-devel/smach_msgs/msg/SmachContainerStatus.msg,

Reply:
Wow, what is this service, I didn't notice this because this service is not used in scripts of `jsk_panda_teleop`...

I found this is used in @ykawamura96 's demo code:
https://gitlab.jsk.imi.i.u-tokyo.ac.jp/y-kawamura/eel_bowl/-/blob/90b2273a87456adb7b6ec6cff66af4000a62da2e/eel_bowl_core/ros_scripts/imitation_agent_server.py#L13
https://gitlab.jsk.imi.i.u-tokyo.ac.jp/y-kawamura/eel_bowl/-/blob/90b2273a87456adb7b6ec6cff66af4000a62da2e/eel_bowl_core/ros_scripts/imitation_agent_server.py#L32
Maybe he forgot to use
https://gitlab.jsk.imi.i.u-tokyo.ac.jp/y-kawamura/eel_bowl/-/blob/90b2273a87456adb7b6ec6cff66af4000a62da2e/srv/CommandImitationAgent.srv
and delete CommandImitationAgent.srv in `jsk_panda_teleop`.

So we have to...
1. Fix @ykawamura96 's code to use CommandImitationAgent.srv in his repository
2. Learn his system and add comment to CommandImitationAgent.srv in his repository
3. Delete CommandImitationAgent.srv in `jsk_panda_teleop`.

Reply:
CommandImitationAgent.srv is moved to https://gitlab.jsk.imi.i.u-tokyo.ac.jp/y-kawamura/eel_bowl/-/merge_requests/1